src/modules/catalog.js

```diff
@@ -1030,21 +1049,29 @@ class Catalog {
    * Retrieves item groups from index
    *
    * @function getItemGroups
+   * @param {object} parameters - Additional parameters for item group details
+   * @param {string} [parameters.section="Products"] - The section of the index to use. Defaults to Products.
    * @param {object} [networkParameters] - Parameters relevant to the network request
    * @param {number} [networkParameters.timeout] - Request timeout (in milliseconds)
    * @returns {Promise}
    * @see https://docs.constructor.com/reference/catalog-item-groups
    * @example
    * constructorio.catalog.getItemGroups();
    */
-  getItemGroups(networkParameters = {}) {
+  getItemGroups(parameters = {}, networkParameters = {}) {
```

Comment:
This is a breaking change, along with the one on line[1241](https://github.com/Constructor-io/constructorio-node/pull/226/files#diff-6f411272c67708d515331aee39f1e4265fb0eb247447119e3ac65af0236d48afR1241). Should I update the minor version?